google_helpers/bigquery/bq_support.py

```diff
@@ -497,7 +497,7 @@ def build_bq_filter_and_params(filters, comb_with='AND', param_suffix=None, with
                     query_param['parameterType']['type'] = ('STRING' if re.compile(ur'[^0-9\.,]', re.UNICODE).search(values[0]) else 'INT64')
                     query_param['parameterValue']['value'] = values[0]
                     if query_param['parameterType']['type'] == 'STRING' and '%' in values[0]:
-                        filter_string += "{}{} LIKE @{}".format('' if not field_prefix else field_prefix, attr, param_name)
+                        filter_string += "LOWER({}{}) LIKE LOWER(@{})".format('' if not field_prefix else field_prefix, attr, param_name)
```

Comment:
There's no need for LOWER({}{} because that's the prefix and name of the field--those should always be passed in correctly, but also, BQ isn't case sensitive in column/field names, or alias names.